traffic_editor/thumbnail_generator/README.md

```diff
@@ -49,15 +49,15 @@ The orthogonal views of the models with a green screen will have been saved in t
 ./scripts/start_gazebo_screen.sh white
 
 # in a second terminal
-./scripts/top_view_generator.py -m test/model_list.yaml -o images/white
+./scripts/top_view_generator.py test/model_list.yaml images/white
 ```
 
 Similarly, the folder `images/white/` will have been populated when the script finishes.
 
 The contours of the models are then extracted from the green-screened images using OpenCV to generate a transparency mask, used for cropping the white-background images, so that it stays centered and is as small as possible. This can be done by calling,
 
 ```bash
-./scripts/crop.py -m test/model_list.yaml -g images/green -w images/white -o images/cropped
+./scripts/crop.py test/model_list.yaml images/cropped -g images/green -w images/white
```

Comment:
the `-o` argument is still required in the source code, were you intending to remove it?